lib/components/hike_screen_widget.dart

```diff
@@ -244,14 +245,14 @@ class HikeScreenWidget {
                       trailing: model.hikers[index].id == model.beacon.leader.id
                           ? GestureDetector(
                               onDoubleTap: () {
```

Comment:
It would be better to add some instructions for leader, like "Double tap on change the duration".
Also this gesture detector instead of being at the trailing part of the tile should be on the entire tile. It took me about a min to figure why tapping is not displaying the dialog box or we can just change it to a more accessible widget like a button.

Reply:
Yes we can add a FAB, that will open a dialogue box.